### PR TITLE
Feature/blue 1210 update commands

### PR DESCRIPTION
**Pull request by coutinol**
(/) tests are passing

### CODE REVIEW

File: helper/general_objects/models.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ type Pageable struct {
 	Unpaged    bool   `json:"unpaged"`
 }
 
-type Tag struct {
+type KeyValue struct {
```
Comment (coutinol) [inline helper/general_objects/models.go:23]: renaming as we use the same structure for transmission metadata (which are not Tags)

File: testing/asset/properties/main.tf [hunk 2]
```diff
@@ -208,4 +208,8 @@ resource "leanspace_properties" "geopoint_node_property" {
 
 output "all_properties" {
   value = data.leanspace_properties.all
+}
+
+output "test_numeric_property" {
```
Comment (coutinol) [inline testing/asset/properties/main.tf:213]: was a bug already present in master branch

File: services/commands/release_queues/schemas.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+		Description:  "What transformation strategy shall be applied on created and updated Commands",
+	},
+	"command_transformer_plugin_configuration_data": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Configuration data used by the Command Transformer Plugin (coming soon)",
+	},
+	"global_transmission_metadata": general_objects.KeyValuesSchema,
+	"logical_lock": {
+        Type:     schema.TypeBool,
```
Comment (jerome-leanspace) [inline services/commands/release_queues/schemas.go:47]: mix of tabs and spaces?

File: services/commands/release_queues/parsers.go [hunk 4]
```diff
@@ -0,0 +1,49 @@
+package release_queues
```
Comment (jerome-leanspace) [inline services/commands/release_queues/parsers.go:1]: So everything works even if some field are not provided, right? (since I saw some are optional)

**Review by jerome-leanspace — COMMENTED** [review]
small comments
I haven't tested it yet, will do that on Monday

File: services/commands/release_queues/schemas.go (hunk 3 above)
Comment (coutinol) [reply services/commands/release_queues/schemas.go:47]: I'll check that

File: services/commands/release_queues/parsers.go (hunk 4 above)
Comment (coutinol) [reply services/commands/release_queues/parsers.go:1]: yep, the following release queue was created in yuri with terraform script: 96e09c49-e8b0-4352-9df7-62066b339802

**Review by jerome-leanspace — COMMENTED** [review]
Left 1 extra comment for validation
Also for the formatting I would recommand to do `gofmt -w -s .` which will format everything

File: services/commands/release_queues/schemas.go [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+	"command_transformer_plugin_id": {
+		Type:         schema.TypeString,
+		Optional:     true,
+		ValidateFunc: validation.IsUUID,
+		Description:  "The Id of the Command Transformer Plugin",
+	},
+	"command_transformation_strategy": {
+		Type:         schema.TypeString,
+		Optional:     true,
+		Description:  "What transformation strategy shall be applied on created and updated Commands",
```
[the remark this answers is not in the document]
Comment (jerome-leanspace) [reply services/commands/release_queues/schemas.go:38]: yes like in here: https://github.com/leanspace/terraform-provider-leanspace/blob/main/services/agents/remote_agents/schemas.go#L75

**Review by jerome-leanspace — APPROVED** [review]
🎉

**Pull request merged**